src/Microsoft.Health.Dicom.Api/Controllers/RetrieveController.cs

```diff
@@ -33,6 +33,7 @@ namespace Microsoft.Health.Dicom.Api.Controllers;
 [QueryModelStateValidator]
 [ServiceFilter(typeof(DicomAudit.AuditLoggingFilterAttribute))]
 [ServiceFilter(typeof(PopulateDataPartitionFilterAttribute))]
+[ServiceFilter(typeof(PopulateOriginalRequestFilterAttribute))]
```

Comment:
Do we want to support get Original on all sub-resources? I thought the requirement was only on WADO resource for audit purpose.

Reply:
Moved from class to individual method